### PR TITLE
tt-rss-plugin-fever: Init at 2.2.0

### PR DESCRIPTION
**Pull request by Madahin**
###### Motivation for this change

Added [tinytinyrss-fever-plugin ](https://github.com/DigitalDJ/tinytinyrss-fever-plugin)

This is a plugin to add a fever endpoint to tiny tiny rss. It is usefull for some agregator like [NewsFlash](https://gitlab.com/news-flash/news_flash_gtk).

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ajs124** [comment]
Supersedes #65532

File: pkgs/servers/tt-rss/plugin-fever/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  meta = with stdenv.lib; {
+    description = "Allows Fever compatible RSS clients to use Tiny Tiny RSS";
+    longDescription = ''
+    This is an open source plugin for Tiny Tiny RSS which simulates the Fever API.
+    This allows Fever compatible RSS clients to use Tiny Tiny RSS.
+    end-point URL is https://example.com/tt-rss/plugins.local/fever/
+    '';
+    license = licenses.gpl3;
+    homepage = "https://github.com/DigitalDJ/tinytinyrss-fever-plugin";
+    maintainers = with maintainers; [ madahin ];
```
Comment (SuperSandro2000) [inline pkgs/servers/tt-rss/plugin-fever/default.nix:27]: Please put this into a separate commit with the message ``maintainers: add madahin``.

File: pkgs/servers/tt-rss/plugin-fever/default.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    #sha256 = "0bylgm9w8rxgrgsh2zrq2wsjy3ff1mz09ad5zmv7pn547cldhqv6";
+    sha256 = "000ys6r6a90x1v6fjyb86b0aab6j7g2d29w8d5sccyv895m4x8r1";
+  };
+
+  installPhase = ''
+    mkdir -p $out/fever
+    cp *.php $out/fever
+  '';
+
+  meta = with stdenv.lib; {
```
Comment (SuperSandro2000) [inline pkgs/servers/tt-rss/plugin-fever/default.nix:18]:
```suggestion
  meta = with lib; {
```

File: pkgs/servers/tt-rss/plugin-fever/default.nix [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    description = "Allows Fever compatible RSS clients to use Tiny Tiny RSS";
+    longDescription = ''
+    This is an open source plugin for Tiny Tiny RSS which simulates the Fever API.
+    This allows Fever compatible RSS clients to use Tiny Tiny RSS.
+    end-point URL is https://example.com/tt-rss/plugins.local/fever/
+    '';
+    license = licenses.gpl3;
```
Comment (SuperSandro2000) [inline pkgs/servers/tt-rss/plugin-fever/default.nix:25]: Only or Plus?

File: pkgs/servers/tt-rss/plugin-fever/default.nix [hunk 4]
```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, ... }: stdenv.mkDerivation rec {
+  pname = "tt-rss-plugin-fever";
+  version = "2.2.0";
+
+  src = fetchFromGitHub {
+    owner = "DigitalDJ";
+    repo = "tinytinyrss-fever-plugin";
+    rev = "${version}";
```
Comment (SuperSandro2000) [inline pkgs/servers/tt-rss/plugin-fever/default.nix:8]:
```suggestion
    rev = version;
```

File: pkgs/servers/tt-rss/plugin-fever/default.nix [hunk 5]
```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, ... }: stdenv.mkDerivation rec {
+  pname = "tt-rss-plugin-fever";
```
Comment (SuperSandro2000) [inline pkgs/servers/tt-rss/plugin-fever/default.nix:2]:
```suggestion
{ stdenv, fetchFromGitHub }:

stdenv.mkDerivation rec {
  pname = "tt-rss-plugin-fever";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
Closing due to inactivity from author.
